torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -640,6 +640,11 @@ std::shared_ptr<SugaredValue> toSugaredValue(
   }
 
   if (py::isinstance<py::function>(obj)) {
+    if (py::cast<bool>(
+            py::module::import("torch.jit").attr("_is_global_builtin")(obj))) {
+      // See [python globals]
```

Comment:
Can you add a little bit more than `See [python globals]`, maybe, "don't attempt to compile python builtin" or something  ?